### ISSUE TITLE
Rename mesh files in Rajagopal model

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/opensim-models#134]
Mesh file names in Rajagopal model, increasingly used, are out of sync. with the geometry files we distribute resulting in issues with either missing visuals or having to carry around Geometry folder. This seems unnecessary and error prone.

**Comment by jenhicks** [comment]
Make sure that the different namings are not intentional --- in some cases @apoorvar may have intentionally used different names so as not to pull in vtp files from the distribution that display poorly

**Comment by apoorvar** [comment]
Yes, there are definitely some instances of different geometry files for
the model than the standard geometry files in the opensim distribution. I
remember the pelvis and tibia vtp files had different geometry for sure,
can't remember what/if anything else.

**Comment by aymanhab** [comment]
I believe this has been fixed for the Rajagopal_Opensense model we distribute which has no Geometry folder, however the original separately maintained model uses different geometry file names and needs updating.
